package.json

```diff
@@ -110,6 +111,7 @@
   },
   "build": {
     "appId": "github.com/jgunzelman88/PokeTrax",
+    "buildDependenciesFromSource": true,
```

Comment:
@jgunzelman88 Added this configuration just force the builder to build rather than try (and fail a bunch of times) to pull a prebuilt binary that doesn't exist from better-sqllite3. Hopefully when https://github.com/WiseLibs/better-sqlite3/issues/830 resolves we can use the prebuilt binary.